ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+GCP_BASE_DOMAIN="origin-ci-int-gce.dev.openshift.com"
+GCP_PROJECT="openshift-gce-devel-ci"
+GCP_REGION="us-east1"
+
+cat >> "${CONFIG}" << EOF
+baseDomain: ${GCP_BASE_DOMAIN}
+compute:
+- name: worker
+  replicas: 3
```

Comment:
is this necessary? i think we can actually leave this out and installer just adds these automatically by default..

the aws and vpshere ones also don't add this

Reply:
I had added it so the python script used to modify the file would match what is in our docs. I'll modify this script instead add the "compute" section.